app/views_redesigned/properties/_layout.html.erb

```diff
@@ -0,0 +1,32 @@
+<% unless local_assigns[:override]&.include? :page_heading %>
+  <% content_for :page_heading do %>
+    <%= page_heading "Viewing", "Properties",
+        subtitle: property.name,
+        icon: "fas fa-tasks" %>
```

Comment:
Icon number 2. We talked about this briefly. Not sure which one we want to go with.